modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -21,8 +21,8 @@ The {pipelines-title} Operator now provides the option to choose the components
 The supported profiles are:
 
 * Lite: This installs only Tekton Pipelines.
-* Basic: This installs Tekton Pipelines and Tekton Triggers.
-* All: This is the default profile used when the `TektonConfig` CR is installed. This profile installs all of the Tekton components: Tekton Pipelines, Tekton Triggers, Tekton Addons (which include `ClusterTasks`, `ClusterTriggerBindings`, `ConsoleCLIDownload`, `ConsoleQuickStart` and `ConsoleYAMLSample` resources).
+* Basic: This installs Tekton Pipelines, Tekton Triggers, and Tekton Chains.
+* All: This is the default profile used when the `TektonConfig` CR is installed. This profile installs all of the Tekton components: Tekton Pipelines, Tekton Triggers, Tekton Chains, {pac}, and Tekton Addons (which include `ClusterTasks`, `ClusterTriggerBindings`, `ConsoleCLIDownload`, `ConsoleQuickStart` and `ConsoleYAMLSample` resources).
```

Comment:
In each module, CR needs to be defined in the first use instance, i.e. "custom resource (CR)"